### ISSUE TITLE
PHP 8.1 compatibility

### ISSUE THREAD

**Issue by peterjaap** [mollie/magento2#511]
Running into this error when adding a product to the cart;

```
[2022-04-12T18:37:59.118868+00:00] .CRITICAL: Exception: Deprecated Functionality: str_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated in /data/pluim/magento2/vendor/mollie/m
agento2/Config.php on line 631 in /data/pluim/magento2/vendor/magento/framework/App/ErrorHandler.php:61
```

See https://github.com/mollie/magento2/pull/510 for a possible fix.

**Comment by Frank-Magmodules** [comment]
Hi @peterjaap , thank you for opening this issue and the [PR](https://github.com/mollie/magento2/pull/510). 
We will be doing some final Magento 2.4.4 / PHP 8.1 checks today and push for an update tomorrow!

**Comment by Frank-Magmodules** [comment]
Hi @peterjaap,

We are happy to share that we've just released the new [2.10.0](https://github.com/mollie/magento2/releases/tag/v2.9.0) version where we have added the compatibility for Magento 2.4.4 / PHP 8.1. Thank you for opening this issue and for your PR contribution! 

We are closing this issue now but please feel free to reopen the issue if this still occurs.
